### DISCUSSION TITLE
Plotfile in ReactEval

### DISCUSSION THREAD

**Discussion by sachdevakunal** [AMReX-Combustion/PelePhysics#544] [Q&A]
Hi. I am working with the test case ReactEval in PelePhysics, and I want plot files after every 20 time steps. I am running the test case for 500 time steps. Currently, it outputs 2 plotfiles, one at the start and one at the end. I tried adding the line amr.plot_int = 20 (as in PeleLMeX), but it's not working.
Can anyone help me with this?

**Comment by baperry2** [comment]
Unfortunately, the ReactEval utility does not support dumping plot files at intermediate states, only at its start and end. It was written to be a simple test for reaction capability in PelePhysics and more general capability has not been added. You'd have to do some nontrivial code rearrangement to enable plotting at intermediate times, which would involve pulling the timestepping loop outside of the MFIter loop and then moving the plotting routine to the end of the timestep rather than the end of the program as a whole.

**Comment by sachdevakunal** [comment]
Okay got it. Thanks for your quick response.
